source/reference/minio-mc/mc-idp-ldap-accesskey-edit.rst

```diff
@@ -0,0 +1,165 @@
+.. _minio-mc-idp-ldap-accesskey-edit:
+
+==============================
+``mc idp ldap accesskey edit``
+==============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+.. mc:: mc idp ldap accesskey edit
+
+
+Description
+-----------
+
+.. start-mc-idp-ldap-accesskey-edit-desc
+
+:mc:`mc idp ldap accesskey edit` modifies the specified access key on the local server.
```

Comment:
```suggestion
:mc:`mc idp ldap accesskey edit` modifies the specified access key on the MinIO deployment.
```

Reply:
same comment w.r.t. linking to access keys and noting this only works on ad/ldap user access keys

Reply:
👍